### ISSUE TITLE
Validation loss : deployed vs full model

### ISSUE THREAD

**Issue by valentinFnr** [shoutOutYangJie/MobileOne#2]
Hi, 
Thank you very much for the implementation. 
I would like to know if you still have the graphs of the full model validation loss and the inference (ready to deploy) model validation loss. 
I would like to know how both models behaved in the very early stage of training (first 20 epochs). 
Thank you very much for considering my request :)

**Comment by shoutOutYangJie** [comment]
I provide the first 20 epochs' accuracy from the training logger.
Both models should have the same performance because the weight should be equivalent. I have checked my model， using a random input and random weight. You can try it by yourself.  Model output should be same.
mobileone_s0, epoch 0, acc 11.02199935913086
mobileone_s0, epoch 1, acc 25.20800018310547
mobileone_s0, epoch 2, acc 34.14400100708008
mobileone_s0, epoch 3, acc 39.06599807739258
mobileone_s0, epoch 4, acc 40.4839973449707
mobileone_s0, epoch 5, acc 42.15599822998047
mobileone_s0, epoch 6, acc 44.534000396728516
mobileone_s0, epoch 7, acc 44.3120002746582
mobileone_s0, epoch 8, acc 46.37199783325195
mobileone_s0, epoch 9, acc 45.077999114990234
mobileone_s0, epoch 10, acc 47.44599914550781
mobileone_s0, epoch 11, acc 48.06999969482422
mobileone_s0, epoch 12, acc 48.4119987487793
mobileone_s0, epoch 13, acc 47.77399826049805
mobileone_s0, epoch 14, acc 48.70399856567383
mobileone_s0, epoch 15, acc 50.86399841308594
mobileone_s0, epoch 16, acc 40.395999908447266
mobileone_s0, epoch 17, acc 50.52399826049805
mobileone_s0, epoch 18, acc 48.99399948120117
mobileone_s0, epoch 19, acc 50.77799987792969
mobileone_s0, epoch 20, acc 49.92399978637695

**Comment by valentinFnr** [comment]
Thank you very much for your reply
